File: Difficulty: Easy/Second Largest/second-largest.java
class Solution {
    public int getSecondLargest(int[] arr) {
        // code here
        
        int largest = arr[0];
        int second = -1;
        
        for(int num : arr){
            if(num > largest){
                second = largest;
                largest = num;
            }else if(num > second && num < largest){
                second = num;
            }
        }
        return second;
    }
}

File: Difficulty: Easy/Second Largest/second-largest.py
class Solution:
    def getSecondLargest(self, arr):
        # Code Here
        largest = arr[0]
        sec = -1
        
        for num in arr:
            if num > largest:
                sec = largest
                largest = num
            elif num > sec and num != largest:
                sec = num
                
        return sec